proposals/0000-initial-extension-categorization.md

```diff
@@ -0,0 +1,240 @@
+| MultiWayIf                 | Stable         | Stable                  |
+| NamedFieldPuns             | Stable         | Stable                  |
+| NamedWildCards             | Stable         | Stable                  |
+| NumDecimals                | Stable         | Stable                  |
+| NumericUnderscores         | Stable         | Stable                  |
+| OverloadedLabels           | Stable         | Stable                  |
+| OverloadedLists            | Stable         | Stable                  |
+| OverloadedRecordDot        | Stable         | Stable                  |
+| OverloadedStrings          | Stable         | Stable                  |
+| ParallelListComp           | Stable         | Stable                  |
+| PatternGuards              | Stable         | Stable                  |
+| PatternSynonyms            | Stable         | Stable                  |
+| PolyKinds                  | Stable         | Stable                  |
+| PostfixOperators           | Stable         | Stable                  |
+| QualifiedDo                | Stable         | Stable                  |
+| RankNTypes                 | Stable         | Stable                  |
+| RecordWildcards            | Stable         | Stable                  |
+| RecursiveDo                | Stable         | Stable                  |
+| RoleAnnotations            | Stable         | Stable                  |
+| StandaloneDeriving         | Stable         | Stable                  |
+| StandaloneKindSignatures   | Stable         | Stable                  |
+| StrictData                 | Stable         | Stable                  |
+| TraditionalRecordSyntax    | Stable         | Stable                  |
+| TransformListComp          | Stable         | Stable                  |
+| TupleSections              | Stable         | Stable                  |
+| TypeApplications           | Stable         | Stable                  |
+| TypeFamilies               | Stable         | Stable                  |
+| TypeFamilyDependencies     | Stable         | Stable                  |
+| TypeOperators              | Stable         | Stable                  |
+| TypeSynonymInstances       | Stable         | Stable                  |
+| UnboxedSums                | Stable         | Stable                  |
+| UnboxedTuples              | Stable         | Stable                  |
+| UnicodeSyntax              | Stable         | Stable                  |
+| UnliftedDatatypes          | Stable         | Stable                  |
+| UnliftedFFITypes           | Stable         | Stable                  |
+| UnliftedNewtypes           | Stable         | Stable                  |
+| ViewPatterns               | Stable         | Stable                  |
+
+
+#### 2.1.2 Deprecated Extensions
+
+The following are extensions that are to be categorized as `Deprecated` for both the enabled and `No` versions.
```

Comment:
I still find it slightly confusing that it looks like the default (disabled) state is being deprecated in these cases, although I can see the argument for doing so. Perhaps we could add a sentence or two to emphasize that the deprecation applies to the *flag* `-XNo...` because there is no point in explicitly disabling these extensions, even though all the language editions include equivalent behaviour?

Reply:
This is a great point. I'll add something for this.

Reply:
Thanks for clarifying this @telser, it looks good to me!